ocrd-tool.json

```diff
@@ -18,7 +18,20 @@
       "tags": ["Text recognition and optimization"],
       "description": "Recognize text in lines with tesseract",
       "executable": "ocrd-tesserocr-recognize",
-      "step": "recognition/text-recognition"
+      "step": "recognition/text-recognition",
+      "parameter": {
+        "textequiv_level": {
```

Comment:
I am not sure here: Shouldn't we propagate the `textequiv` contents through all the levels?

Reply:
Yes, `TextEquiv` should be propagated where possible. But you might want to run the recognition on sub-regions, on different levels. If the input PAGE is already zoned on line level, you can (as you implemented) set the PSM to `SINGLE_LINE`. Or you might have an OCR result on the word/glyph level as input and want to recognize only the words to compare results.

Maybe `recognition-level` is less confusing?